mistletoe/base_renderer.py

```diff
@@ -144,24 +144,62 @@ def render_raw_text(self, token):
         """
         return token.content
 
-    def __getattr__(self, name):
```

Comment:
I highly doubt that somebody is relying on this for yet-to-be-introduced token types. And if that were the case it should be easy to adapt.

I think when providing a class to be extended it's very important to define a proper API, i.e. defining the  methods implicitly instead of using such `__getattr__` hacks.

Reply:
OK, thanks for your feedback. Let's remove the method then. If someone won't like it, they will hopefully let us know.